upload/admin/model/extension/payment/coingate.php

```diff
@@ -12,6 +12,12 @@ public function install() {
       ) ENGINE=MyISAM DEFAULT COLLATE=utf8_general_ci;
     ");
 
+    // Modify the currency table to allow 4-character currency code
+    $this->db->query("
+      ALTER TABLE `" . DB_PREFIX . "currency`
+      MODIFY `code` VARCHAR(4) COLLATE utf8_general_ci NOT NULL;
+    ");
+
```

Comment:
I think a check if table is 3 charters should be done before editing table

Reply:
I can add it if you think it's needed, but "modifying" it from 4 to 4 simply won't make any changes, so I thought it wasn't necessary.

Reply:
Commit message got messed up, but I've added the check